cmd/gapir/cc/main.cpp

```diff
@@ -76,6 +77,190 @@ std::vector<uint32_t> memorySizes {
       128 * 1024 * 1024U,       // 128MB
 };
 
+struct Options {
```

Comment:
This Change is probably not the place to fix this, since I think it is a holdover from an earlier time. But none of these variables follow the Google C++ style guide (which we are at least TRYING to follow).

Reply:
https://google.github.io/styleguide/cppguide.html#General_Naming_Rules

Reply:
Yeah, I moved the code. The names are as they were when I encountered the source.